bank/src/main/java/com/clouway/bricky/adapter/http/AmountDTO.java

```diff
@@ -0,0 +1,13 @@
+package com.clouway.bricky.adapter.http;
+
+/**
+ * @author Marian Zlatev <mzlatev91@gmail.com>
+ */
+class AmountDTO {
+
+  public double amount;
```

Comment:
I agree about the public modifier.
I will change it to Double and will read about javax.validation.

Currently I can not make the field final because the transporter class that we use does not pass parameters through constructor, instead a setter method is used and thus final is not permitted.